static/elements/chromedash-featurelist.html

```diff
@@ -152,18 +164,41 @@
 
       // Returns the closest feature <li> to the scroll top position passed in.
       featureInView: function(containerScrollTop) {
-        var closest = null;
-        for (var i = 0, li; li = featureLiList_[i]; ++i) {
-          var dist = li.offsetTop - containerScrollTop;
-          if (dist < 0) {
-            dist = -dist * 1.5;
-          }
-          if (closest == null || dist < closest.dist) {
-            closest = this.filtered[i];
-            closest.dist = dist;
-          }
+
+        // Generate list of features below current scroll position.
+        var items = featureLiList_.filter(function(el) {
+          if (el.offsetTop < containerScrollTop) { return el; }
+        });
+
+        // Grab the last feature (it's at the top of the scroll container).
+        var topmostFeature = items.length ? items[items.length - 1] :
+                                            featureLiList_[0];
+
+        var idx = featureLiList_.indexOf(topmostFeature);
+         // Use next feature in list so milestone labels line up better.
+        var feature = this.features[idx + 1];
+
+        var label = feature.shipped_milestone || feature.impl_status_chrome;
```

Comment:
I am wondering if this will ever bite us due to the loose truthey check.. I think we should be ok just something to think on